change/@fluentui-react-positioning-86dcd6cd-76f3-4f49-b446-04e827493aa7.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28689&commitId=6739c3d0c58ee5569a53eecf6679df20353b65df&env=prod&clientType=FLUENTUI)


  <details><summary><b>Positioning</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Positioning.auto size with async content.floating element is within the boundary.chromium.png|261| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
Screenshot changes because:

old screenshot - resetMaxSize causes lifecycle to reach maximum. Then floating ui runs each middleware just once.

new screenshot - resetMaxSize resets lifecycle only once. Then when maxSize finishes for the first time, it concluded that size has changed (because border-box is applied which causes floatingui to calculate size differently). maxSize resets the lifecycle again and result in the floating element 1px larger.